sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DistroHelper.java

```diff
@@ -527,12 +548,15 @@ public Properties getFrontendProperties(Server server) throws MojoExecutionExcep
 	}
 
 	public Properties getFrontendPropertiesForServer(Artifact artifact, File directory) throws MojoExecutionException {
-		if (new Version(artifact.getVersion()).higher(new Version("3.0.0-beta.16"))) {
-			return getFrontendProperties(artifact, directory);
-		} else {
-			return PropertiesUtils.getFrontendPropertiesFromSpaConfigUrl(
-					"https://raw.githubusercontent.com/openmrs/openmrs-distro-referenceapplication/" + artifact.getVersion() + "/frontend/spa-build-config.json");
+		if (artifact.getArtifactId().equals("referenceapplication-distro")) {
```

Comment:
This doesn't work for all cases.  Specifically, I was testing with a refapp 2.x distro as the parent, and in trying to do that it was failing here, as no resource in github exists for spa-build-config.json in a 2.x release/tag of the refapp, as far as I can tell.

So I wrapped this in a check to only resolve these properties if we are dealing with the "referenceapplication-distro" artifact, which is what is used starting in refapp 3.x.  Can you confirm this is what we want @ibacher  and @dkayiwa ?  This whole downloading frontend config from github is...odd.